docs/wiki/Whats-new.md

```diff
@@ -62,6 +63,14 @@ This article will be updated as and when changes are made to the above and anyth
 
 Here's what's changed in Enterprise Scale/Azure Landing Zones:
 
+### 🔃 Policy Refresh H1 FY26
+
+- *New* custom policy [Audit-AKS-kubenet](https://www.azadvertizer.net/azpolicyadvertizer/Audit-AKS-kubenet.html) to detect AKS clusters using the deprecated 'kubenet' network plugin. This policy is included in the initiative [Enforce-Guardrails-Kubernetes](https://www.azadvertizer.net/azpolicyinitiativesadvertizer/Enforce-Guardrails-Kubernetes.html) with the effect defaulted to "Audit". The effect can be changed via the new parameter `akaKubenetEffect`. Added as the deprecation of kubenet is planned for 31 March 2028. For more information review [https://learn.microsoft.com/en-us/azure/aks/configure-kubenet](https://learn.microsoft.com/en-us/azure/aks/configure-kubenet).
```

Comment:
Lets add a link to the kubenet announcement to help folks understand the why https://azure.microsoft.com/en-gb/updates?id=485172

Also lets remind folks where its assigned in the hierarchy